spec/models/piece_spec.rb

```diff
@@ -1,5 +1,226 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "is_obstructed? returns false if path is not blocked" do
```

Comment:
We weren't really sure if checking for out of bounds, start  = destination, or move <> vertical/horizontal/diagonal should belong to a method called is_obstructed. if we start checking for vertical/horizontal/diagonal don't we have to also start coding to allow for the knight? or create a separate is_obstructed? for the knight class.